examples/prophet/train.py

```diff
@@ -0,0 +1,86 @@
+# The data set used in this example is from 'https://raw.githubusercontent.com/facebook/prophet/master/examples/example_wp_log_peyton_manning.csv'
+
+import os
+import warnings
+import sys
+
+import pandas as pd
+import numpy as np
+
+import mlflow
+import mlflow.pyfunc
+
+import cloudpickle
+
+import fbprophet
+from fbprophet import Prophet
+from fbprophet.diagnostics import cross_validation
+from fbprophet.diagnostics import performance_metrics
+
+class FbProphetWrapper(mlflow.pyfunc.PythonModel):
+
+    m = None
+
+    def __init__(self, model):
+        m = model
+        super(FbProphetWrapper, self).__init__()
+
+
+    def load_context(self, context):
+        from fbprophet import Prophet
+        return
+
+    def predict(self, context, model_input):
+        future = m.make_future_dataframe(periods=model_input['periods'])
+        return (m.predict(future), m)
```

Comment:
@dr3s Have you tested serving this model via `mlflow models serve`? I suspect that inference won't work because the model returns two outputs: the model predictions and the underlying Prophet model. Is it possible to make this model conform to the pyfunc inference API?: https://mlflow.org/docs/latest/python_api/mlflow.pyfunc.html#inference-api (e.g., by removing the second output).

Reply:
No, thanks for pointing that out.  I guess I should expand the example to include predict